src/ValueSchema/AbstractValueSchema.php

```diff
@@ -10,10 +10,16 @@ abstract class AbstractValueSchema implements ValueSchemaInterface
 
     protected ?string $description;
 
-    public function __construct(bool $nullable = false, ?string $description = null)
+    /**
+     * @var ?mixed
+     */
+    protected $example;
```

Comment:
Tohle neni spolecne pro vsechna schemata, kazde schema ma example jineho typu. Plati i pro buildery, tim padem to znamena, ze ta validace je na spravnem miste, jenom ne ve spolecne metode, ale v metodach pro jednotlive buildery. Druha moznost, pokud si cheme usetrit praci, je umoznit dat jakykoliv example kamkoliv, ale v tom pripade, bych se na validaci vykaslal uplne, protoze je to pak podivny mezistav.